src/main/java/uk/gov/mca/beacons/api/exceptions/RestResponseEntityExceptionHandler.java

```diff
@@ -0,0 +1,44 @@
+package uk.gov.mca.beacons.api.exceptions;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpHeaders;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.MethodArgumentNotValidException;
+import org.springframework.web.bind.annotation.ControllerAdvice;
+import org.springframework.web.context.request.WebRequest;
+import org.springframework.web.servlet.mvc.method.annotation.ResponseEntityExceptionHandler;
+import uk.gov.mca.beacons.api.services.GetValidationErrorResponse;
+
+@ControllerAdvice
```

Comment:
Global error handler for all controllers, see [this article for more information](https://www.baeldung.com/exception-handling-for-rest-with-spring#controlleradvice).

We can exclude certain controllers from using this handler, i.e: if we just wanted this on the owner controller, but I couldn't think of a reason why we would not want the handling of 4xx errors through this component.

Currently, the exception handler for controllers is only handling 400 errors but we can extend it over time to also handle other 4xx or 5xx errors. This felt like the smallest piece of code to write to get the errors bound to the body of the response from the controller.